.github/workflows/test.yml

```diff
@@ -23,3 +23,26 @@ jobs:
 
       - name: Run tests
         run: tox -e ${{matrix.python-version}} -- -vrsx --color=yes
+
+  build-windows:
+    name: Test Windows build
+    runs-on: windows-latest
+    strategy:
+      max-parallel: 100
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - name: Checkout repos
+      uses: actions/checkout@v1
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Build x86_64 packages
+      shell: cmd
+      run: |
+        pip install wheel || exit /b
+        python setup.py bdist_wheel
```

Comment:
This builds the package, doesn't run any test. I'd like to see some tests running: maybe they won't pass (and they can be skipped) but at least I'd like to see the library not segfaulting on import.

Reply:
Makes sense, I'll add a simple test that may be used on Windows as well.